src/main/java/htsjdk/samtools/DuplicateScoringStrategy.java

```diff
@@ -80,6 +81,8 @@ public static short computeDuplicateScore(final SAMRecord record, final ScoringS
                         score += SAMUtils.getMateCigar(record).getReferenceLength();
                     }
                     break;
+                case RANDOM:
+                    score += (short) Math.sqrt(record.getReadName().hashCode());
```

Comment:
Doh - thanks for pointing that out.  For some reason I was thinking `DuplicateScoringStrategy` _was_ the enum.  Will fix.